buildspec/packageTestVsix.yml

```diff
@@ -23,7 +23,6 @@ phases:
                         echo "CodeArtifact connection failed. Falling back to npm"
                     fi
                 fi
-            # --unsafe-perm is needed because we run as root
             - npm ci --unsafe-perm
```

Comment:
is the comment wrong? or why remove it?

Reply:
Probably removed by accident over the course of several changes. `unsafe-perm` might not actually be needed anymore, but haven't tried it.

Reply:
Codebuild runs as root by default: https://docs.aws.amazon.com/codebuild/latest/userguide/troubleshooting.html#troubleshooting-root-build-commands

Reply:
Was able to remove it and everything works.